src/simpleDetFunc.c

```diff
@@ -224,7 +224,23 @@ void rollSigma(int startIndex, int interval, float scaleFactor,
 
 
 
+#define EXP_A 184
+#define EXP_C 16249 
 
+float EXP_APPROX(float x)
+{
+	//union allows us to treat the same 4 bytes of memory as both a float and 2 shorts
```

Comment:
I would add a a comment providing a source for the algorithm (maybe something like:)

```
// Approximation from Schraudolph (1998)
// "A Fast, Compact Approximation of the Exponential Function"
```